RUFAS/data_validator.py

```
@@ -1796,6 +1797,79 @@ def validate_feedlot_config(feedlot_config: dict[str, Any]) -> None:
+            The beef_cow_calf config block from the animal input file.
+
+        Raises
+        ------
+        ValueError
+            If any of the following rules are violated:
+
+            - ``mature_cow_weight_kg`` must be > 0 and finite.
+            - ``weaning_age_days`` must be > 0 and finite.
+            - ``breeding_season_length`` must be > 0 and finite.
+            - ``natural_service_bull_ratio`` must be between 1 and ``MAX_BULL_TO_COW_RATIO``.
+            - ``breeding_season_start_day`` must be 1–365 when present and non-None.
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)